source/commands/ci/_tests/runner.test.ts

```diff
@@ -16,6 +16,10 @@ const defaultAppArgs: Partial<SharedCLI> = {
 const source = new FakeCI({})
 const platform = new FakePlatform()
 
+beforeEach(() => {
+  mockRunDangerSubprocess.mockReset()
```

Comment:
Currently each assertion here is actually testing against the arguments passed into the _first_ invocation of the danger runner, not the one that they've just performed.

Reply:
👍